### ISSUE TITLE
Support links in <p role="desc"> in script documentation

### ISSUE THREAD

**Issue by bertfrees** [daisy/pipeline-webui#32]
Currently the text is cut off at the first link. E.g.

```html
<p px:role="desc">
    Input HTML. A <a href="http://snaekobbi.github.io/braille-css-spec">braille CSS</a>
    style sheet must be linked from or embedded in the document. See
    <a href="https://code.google.com/p/daisy-pipeline/wiki/ZedAIToPEFDoc#Styling_with_Braille_CSS">examples</a>.
</p>
```
results in "Input HTML. A".

**Comment by josteinaj** [comment]
@bertfrees: I belive there's a px:role="homepage" in case that helps.

**Comment by bertfrees** [comment]
I know, that's useful too, but still. You think the HTML shouldn't be inserted "as is"?

**Comment by josteinaj** [comment]
For the Web UI that would be useful, but I don't know how other clients would cope with tags. I suppose it would be fine. Would the HTML be escaped (with `&lt;`, `&gt;` etc) in the script XML ?

**Comment by bertfrees** [comment]
Ah yes, didn't think of that, I guess other clients would have to ignore the tags somehow. I don't think the HTML should be escaped. But maybe let's wait with this until I have a good use case, and I can use the "homepage" role for now. It works also on options right?

**Comment by josteinaj** [comment]
> It works also on options right?

Yes.

**Comment by josteinaj** [comment]
Actually, no I don't think so, it's for the script description I think, not option/input descriptions.

**Comment by bertfrees** [comment]
Somewhat related: https://github.com/daisy/pipeline-issues/issues/468

**Comment by josteinaj** [comment]
@bertfrees So this issue is blocked on daisy/pipeline-issues#468, right? Or is that a separate issue?

**Comment by bertfrees** [comment]
No, not blocked. 468 was just an idea, a nice to have.

**Comment by josteinaj** [comment]
Ok. But this issue requires a change in the Web API, is there another issue for making this change in the Web API or should I create one?

**Comment by bertfrees** [comment]
No, I haven't made an issue for web API yet.

**Comment by josteinaj** [comment]
blocked on: daisy/pipeline-framework#101
